shell/common/common_benchmarks.sh

```diff
@@ -377,6 +387,32 @@ get_initial_disk() {
   echo -e "$dir"
 }
 
+set_benchmark_data_sizes() {
+  BENCH_DATA_SIZE=$1
+  BENCH_DATA_SIZE_MB=$(( BENCH_DATA_SIZE / 1000000 ))
+  BENCH_DATA_SIZE_GB=$(( BENCH_DATA_SIZE / 1000000000 ))
+  BENCH_DATA_SIZE_TB=$(( BENCH_DATA_SIZE / 1000000000000 ))
+}
```

Comment:
I already said that I don't like the idea of having all these variables, but the way that they're being set is also unintuitive, due to the use of integer division.

Also, the conversion factors are defined elsewhere in `shell/conf/benchmarks_defaults.conf` so I guess it'd make sense to use them here.

Or, just remove this function and replace it with working implementations of `get_bench_data_size_mb`, etc.